CHANGES.txt

```diff
@@ -5,6 +5,7 @@ What's new in Traits UI 4.6.0
 -----------------------------
 
 Fixes
+~~~~~
 
 * Help button now works on Qt (#160)
```

Comment:
Looks like there's some extra indentation here (at least if this is supposed to be reST).